### DISCUSSION TITLE
Report enhancement to add Expand and Collapse feature for Steps and Logs

### DISCUSSION THREAD

**Discussion by amitasil** [ortoniKC/ortoni-report#50] [Ideas]
Hi @ortoniKC, Thanks for such a detailed report!

I want to make an enhancement request. Some of my tests have a big list of steps and console logs and It'll be great if I can able to expand and collapse them.
This will help me not to scroll all the way down my lengthy steps to see my console logs.

**Enhancement Request**:
Add an Expand/Collapse feature by adding an icon to expand and collapse the '**Steps**' and '**Logs**' sections in the Ortoni Report.

Please let me know if you need any further information from my end.

Thanks!

**Comment by ortoniKC** [comment]
![image](https://github.com/user-attachments/assets/49028374-c24d-4bcc-8ae8-e11639da8549)
Instead of expand and collapse I have modified to tabs, it helps I believe.

**Reply by amitasil** [reply]
@ortoniKC This would help for sure. Thanks for making the changes! When will it be available? :)

**Reply by ortoniKC** [reply]
Testing in progress, probably by the end of today

**Reply by ortoniKC** [reply]
The latest version is released. 
Please check.

**Reply by amitasil** [reply]
@ortoniKC Thanks for working on this! This helps now and the report looks easy to read and navigate across different tabs.
